### DISCUSSION TITLE
Renovate dashboard doesn't show anything and only do Python

### DISCUSSION THREAD

**Discussion by sylvestre** [renovatebot/renovate#11370] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Describe the bug

it might be because of the scale of Firefox repository but the dashboard doesn't display anything anymore:
https://app.renovatebot.com/dashboard#github/sylvestre/gecko-dev

Maybe a side effect but only Rust updates are now published:
https://github.com/sylvestre/gecko-dev/pulls

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by viceice** [comment]
You need to enable issues to get the dashboard

**Reply by viceice** [reply]
Renovate doesn't process forks by default

https://docs.renovatebot.com/configuration-options/#includeforks

**Reply by sylvestre** [reply]
Oh, sorry, thanks

**Reply by sylvestre** [reply]
Quick feedback: having an error message or a clue in the dashboard would help to understand the situation btw

**Reply by rarkins** [reply]
Forks are processed if the user does "Select repositories" instead of "All repositories" in the app

**Reply by rarkins** [reply]
If it's skipped because of being a fork then it will definitely be noted in the logs at app.renovatebot.com

**Reply by sylvestre** [reply]
Unfortunately, this isn't the case:
![image](https://user-images.githubusercontent.com/733326/130357048-8f636613-d8b5-41d5-b529-153b5a9ea6b1.png)

**Reply by rarkins** [reply]
If you added it more then 3 days ago then it would not be getting processed any more. Life are only displayed for the last 3 days (I forgot that limitation earlier when commenting)

**Reply by rarkins** [reply]
It's also possible that the repo is timing out due to size, so please let me know if you still suspect that's the case and I'll dig into other logs

**Reply by sylvestre** [reply]
Maybe it was caused by the fact that issues were not enabled. The log part is still empty but I will check later today and tomorrow! 

(it is pretty common that the size of the firefox repo is breaking some tooling :)

**Reply by rarkins** [reply]
I think I may have even tested this repo myself quite a while ago when I saw someone - perhaps yourself - install it. I would not be surprised if its size has exceeded our default limits but would be happy to do my best to get it working one way or another. 

This is likely "App support" (squeezing resources out of the hosted solution and not changes to any OSS code) so best to keep as a Discussion for now anyway.

**Reply by sylvestre** [reply]
Yeah, you had a look back then and it worked fine until April.

The dashboard is still empty:
https://app.renovatebot.com/dashboard#github/sylvestre/gecko-dev

Even if the bot updated this PR:
https://github.com/sylvestre/gecko-dev/pull/459#event-5190866727
(only pin dependencies)

**Reply by viceice** [reply]
Can you open the browser console and check, if there are any failed requests? Can you relogin / clear cache for dashboard?

**Reply by sylvestre** [reply]
Same (i did it before and just did it again :)

**Reply by sylvestre** [reply]
and nothing in the console

**Reply by rarkins** [reply]
It's strange - perhaps unprecedented - that it makes to update a PR but somehow later fails and exits ungracefully without posting its log. Maybe Out of Memory is a possibility.

**Reply by sylvestre** [reply]
Yeah, i can confirm that i am getting pin updates and rust updates but not python's.

**Reply by rarkins** [reply]
Looks like it's timing out after 30 minutes. The Pin updates PR would normally be first in the list of branches, so that makes some sense. I'll try upping the limit to 40 minutes but at this point I'm not 100% sure if it's happily ploughing away before being interrupted after 30 minutes or if there's some aspect of the repo which causes it to essentially die.

**Reply by sylvestre** [reply]
40 minutes didn't change much.
I limited the analysis to only a few directories and it fixed the issue.

**Reply by rarkins** [reply]
@sylvestre FYI I am in planning stages for a new backend which will allow us to control vcpu/ram/disk on a per-job basis, meaning we can decide for important projects to throw CPUs, RAM and disk at the problem. Potentially done by end of year

**Reply by sylvestre** [reply]
Excellent, please let me know if you want me to test it!
